ClientApp/viking-vault/src/ReponseModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+const modalOpenStyle = {
```

Comment:
inline styles are generally best used when you actually need to compute values using JavaScript rather than using values which do not change.